apps/i18n/craft/vi_vn.json

```diff
@@ -1,34 +1,34 @@
 {
   "absolute": "giá trị tuyệt đối",
-  "agentGenericFailureMessage": "Keep coding! Remember to use The Agent to help.",
-  "agentTooFewBlocksFailureMessage": "Try adding just one block at a time to get The Agent to the right spot.",
+  "agentGenericFailureMessage": "Hãy tiếp tục lập trình! Nhớ hãy sử dụng Đặc vụ để giúp đỡ.",
+  "agentTooFewBlocksFailureMessage": "Hãy thử thêm chỉ 1 khối lệnh vào thời điểm Đặc vụ tới được đúng chỗ.",
   "at": "vào",
-  "agentDiamondPathCongrats": "Congratulations, you found the diamond path! You have collected {count, plural, one {1 diamond} other {# diamonds}} so far!",
+  "agentDiamondPathCongrats": "Xin chúc mừng, bạn tìm thấy đường dẫn kim cương! Đến thời điểm hiện tại bạn đã được thu thập {count, plural,one {1 kim cương}other {# kim cương}}!",
```

Comment:
Does the spacing matter?

Reply:
tested manually, looks like it does not!